receiver/prometheusreceiver/internal/metricfamily.go

```diff
@@ -359,3 +372,27 @@ func populateLabelValues(orderedKeys []string, ls labels.Labels) []*metricspb.La
 	}
 	return lvs
 }
+
+func createNodeResource(resourceLabels map[string]string) *resourcepb.Resource {
+	location, ok1 := resourceLabels[k8SKeyLocation]
```

Comment:
Nit: you don't need 3 `ok`s here. Using one `ok` is ok :)

Reply:
ok :)